doc/user/content/overview/architecture.md

```diff
@@ -110,14 +110,12 @@ view is in terms of how long the generated dataflow persists.
   read on this data at any given time than it is to create an answer from
   scratch.
 
-The only "wrinkle" in the above explanation is when you perform reads on views:
-no dataflow gets created, and Materialize instead serves the result from an
-existing dataflow.'
+The only wrinkle in the above explanation is that when you perform reads on views, no dataflow gets created, and Materialize instead serves the result from an
```

Comment:
Technically the "no dataflow gets created" case is slightly more expansive than "perform reads on views". If you select something that only involves maps, filters, and projects on top of a view, we serve it from an existing dataflow

Reply:
And orderby/limit/offsets too! Perhaps best to narrow the explanation down to "in some cases, when the queries are relatively simple, Materialize can avoid creating a dataflow at all" or somesuch?